### DISCUSSION TITLE
Using MadMax as plotter in Machinaris

### DISCUSSION THREAD

**Discussion by sadreddini** [guydavis/machinaris#204] [Q&A] [answered]
Hi,
It might be a silly question but I am not able to find parameter to set MadMax as default plotter or choose it for one plotting session.
Thanks for assistance.

**Comment by guydavis** [comment, marked as answer]
Good day!  Please see this [wiki page](https://github.com/guydavis/machinaris/wiki/MadMax) on Madmax within Machinaris.  

If this answers your question, please mark this Discussion as Answered.  Thanks!

**Reply by sadreddini** [reply]
Thanks a lot!
